repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/gatherinfocontract/GatherTestInformation.java

```diff
@@ -34,23 +35,16 @@
     private Set<FailureLocation> failureLocations;
     private Set<String> failureNames;
     private String failingModulePath;
+    private boolean skipSettingStatusInformation;
 
     private ContractForGatherTestInformation contract;
 
-    public GatherTestInformation(ProjectInspector inspector, ContractForGatherTestInformation contract) {
+    public GatherTestInformation(ProjectInspector inspector, ContractForGatherTestInformation contract, boolean skipSettingStatusInformation) {
```

Comment:
In fact this one is really for you to avoid putting in JobStatus wrong information. So maybe it's enough to put it in the contract, I really don't know. Tell me :)